code/script.js

```diff
@@ -1,325 +1,356 @@
+    },
+    {
+      name: 'Jess',
+      img: 'images/jess.svg',
+      hair: 'black',
+      eyes: 'blue',
+      accessories: ['glasses'],
+      other: []
+    },
+    {
+      name: 'Jocelyn',
+      img: 'images/jocelyn.svg',
+      hair: 'black',
+      eyes: 'brown',
+      accessories: ['glasses'],
+      other: []
+    },
+    {
+      name: 'Jon',
+      img: 'images/jon.svg',
+      hair: 'brown',
+      eyes: 'green',
+      accessories: ['glasses'],
+      other: []
+    },
+    {
+      name: 'Jordan',
+      img: 'images/jordan.svg',
+      hair: 'yellow',
+      eyes: 'hidden',
+      accessories: ['glasses', 'hat'],
+      other: []
+    },
+    {
+      name: 'Josephine',
+      img: 'images/josephine.svg',
+      hair: 'grey',
+      eyes: 'brown',
+      accessories: [],
+      other: []
+    },
+    {
+      name: 'Josh',
+      img: 'images/josh.svg',
+      hair: 'yellow',
+      eyes: 'green',
+      accessories: [],
+      other: []
+    },
+    {
+      name: 'Jude',
+      img: 'images/jude.svg',
+      hair: 'black',
+      eyes: 'green',
+      accessories: [],
+      other: []
+    },
+    {
+      name: 'Julie',
+      img: 'images/julie.svg',
+      hair: 'black',
+      eyes: 'brown',
+      accessories: ['glasses', 'hat'],
+      other: []
+    },
+  ]
 
-// Draw the game board
-const generateBoard = () => {
-  board.innerHTML = ''
-  charactersInPlay.forEach((person) => {
-    board.innerHTML += `
-      <div class="card">
-        <p>${person.name}</p>
-        <img src=${person.img} alt=${person.name}>
-        <div class="guess">
-          <span>Guess on ${person.name}?</span>
-          <button class="filled-button small" onclick="guess('${person.name}')">Guess</button>
+  // Global variables
+  let secret
+  let currentQuestion
+  let charactersInPlay
+  let keep
+  
+  // Draw the game board
+  const generateBoard = () => {
+    board.innerHTML = ''
+    charactersInPlay.forEach((person) => {
+      board.innerHTML += `
+        <div class="card">
+          <p>${person.name}</p>
+          <img src=${person.img} alt=${person.name}>
+          <div class="guess">
+            <span>Guess on ${person.name}?</span>
+            <button class="filled-button small" onclick="guess('${person.name}')">Guess</button>
+          </div>
         </div>
-      </div>
-    `
-  })
-}
+      `
+    })
+  }
 
-// Randomly select a person from the characters array and set as the value of the variable called secret
-const setSecret = () => {
-  secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
-}
+  // Randomly select a person from the characters array 
+  const setSecretCharacter = () => {
+    secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
+  }
 
-// This function to start (and restart) the game
-const start = () => {
-  // Here we're setting charactersInPlay array to be all the characters to start with
+  // This function to start (and restart) the game
```

Comment:
Good that you put generateBoard() setSecretCharacter() here, under "const start", I discovered that the game doesn't work if you place it somewhere else 👍